### PR TITLE
Add the GAV coordinates of the annotations to the documentation

### LINKED ISSUE #3110: The documation on the annotations should state which depdency is required
As user of Spotbugs, who would like to use the annotation of Spotbugs,
I would like to know the artifact coordinates of the jar containing the annotations,
so that I can include them in my project quite easily.

*Acceptance Criterias*
- The artifact coordinates are easy to find on or via the page [_Annotations_](https://spotbugs.readthedocs.io/en/latest/annotations.html) in the documentation.

### PR DESCRIPTION
**Pull request by gtoison**
This should fix #3110

### CODE REVIEW

**Comment by gtoison** [comment]
The updated page is available here: https://spotbugs--3137.org.readthedocs.build/en/3137/annotations.html
I did not write a translation in Japanese or Portuguese because I don't know these languages, but hopefully this Maven or Gradle dependency declarations are self-explanatory

**Review by JuditKnoll — CHANGES_REQUESTED** [review]
I'm not convinced, that this is the best place for these infos. IMO it would be better to have it elsewhere and just refer that in the annotations page.

There are already some info, how these can be used at the [FindBugs -> SpotBugs migration](https://spotbugs.readthedocs.io/en/latest/migration.html#com-google-code-findbugs-findbugs-annotations), but that's a bit different context.

**Comment by gtoison** [comment]
> I'm not convinced, that this is the best place for these infos. IMO it would be better to have it elsewhere and just refer that in the annotations page.
> 
> There are already some info, how these can be used at the [FindBugs -> SpotBugs migration](https://spotbugs.readthedocs.io/en/latest/migration.html#com-google-code-findbugs-findbugs-annotations), but that's a bit different context.

I initially thought about adding this on the README page but I'm worried that some users might think that the annotation library IS spotbugs (and merely add the dependency but wont actually analyze the project).
I think it makes sense to have them on the Annotations page, but let me know if you see a better place

File: docs/annotations.rst [hunk 1]
```diff
@@ -4,6 +4,22 @@ Annotations
...
 
+Adding the annotations library to a Maven project:
+
+.. parsed-literal::
+	
+  <dependency>
+    <groupId>com.github.spotbugs</groupId>
+    <artifactId>spotbugs-annotations</artifactId>
+    <version>\ |release|\ </version>
+  </dependency>
```
[the remark this answers is not in the document]
Comment (JuditKnoll) [reply docs/annotations.rst]: Thanks for letting me know 😄

**Comment by JuditKnoll** [comment]
> > I'm not convinced, that this is the best place for these infos. IMO it would be better to have it elsewhere and just refer that in the annotations page.
> > There are already some info, how these can be used at the [FindBugs -> SpotBugs migration](https://spotbugs.readthedocs.io/en/latest/migration.html#com-google-code-findbugs-findbugs-annotations), but that's a bit different context.
> 
> I initially thought about adding this on the README page but I'm worried that some users might think that the annotation library IS spotbugs (and merely add the dependency but wont actually analyze the project). I think it makes sense to have them on the Annotations page, but let me know if you see a better place

I was first thinking on the Using the SpotBugs Maven / Gradle Plugin pages, but you are right, that could cause confusion. I can't find any better place, it's okay like this.

**Review by JuditKnoll — APPROVED** [review]

**Pull request merged**